<launch>
  <!-- Create namesapce to avoid naming collisions if launching several drivers -->
  <group ns="hub_1">
    <node pkg="teraranger_array" type="teraranger_one" name="hub_parser" output="screen" required="True" >
      <!-- Set the serial port name -->
      <param name="portname" value="/dev/ttyACM0" />
      <!-- Set the sensor mode 0=precise, 1=Fast, 2=Outdoor-->
      <param name="Mode" value='0' />
      <!-- Remapping topic when using converter nodes-->
      <remap from="ranges" to="ranges_raw" />
    </node>
  </group>
</launch>
